backend/src/plugins/kube.ts

```diff
@@ -103,3 +112,23 @@ const getCurrentNamespace = async () => {
     }
   });
 };
+
+const getCurrentToken = async (currentUser: User) => {
+  return new Promise((resolve, reject) => {
+    if (currentContext === 'inClusterContext') {
+      fs.readFile(
+        // '/var/run/secrets/kubernetes.io/serviceaccount/token',
+        currentUser?.authProvider?.config?.tokenFile,
```

Comment:
@LaVLaS what's the fallback if this is not found?

Reply:
I'm not sure what we could do...

Reply:
would this be better?
```
      const location =
        currentUser?.authProvider?.config?.tokenFile ||
        '/var/run/secrets/kubernetes.io/serviceaccount/token';
      fs.readFile(location, 'utf8', (err, data) => {
        if (err) {
          resolve('');
        }
        resolve(data);
      });
```

Reply:
Any reason we don't try to just fileread the path? I get it is not ideal, but it seems to work... no? Or am I missing something? @LaVLaS @cfchase

Reply:
Ah -- we messaged at the same time, yes that was my thought